contracts/src/PBH4337Module.sol

```diff
@@ -5,6 +5,7 @@ import {Safe4337Module} from "@4337/Safe4337Module.sol";
 import {PackedUserOperation} from "@account-abstraction/contracts/interfaces/PackedUserOperation.sol";
 import {ValidationData} from "@account-abstraction/contracts/core/Helpers.sol";
 import {_packValidationData} from "@account-abstraction/contracts/core/Helpers.sol";
+import {SafeModuleSignatures} from "./helpers/SafeModuleSignatures.sol";
```

Comment:
If this lib consists o one function, maybe we should add it directly to the `PBH4337Module`. Im fine with keeping it as a lib as well if you think its applicable for other contracts though.